tensorflow_recommenders_addons/dynamic_embedding/python/ops/tf_save_restore_patch.py

```diff
@@ -527,10 +539,86 @@ def restore(self, sess, save_path):
     except errors.InvalidArgumentError as err:
       # There is a mismatch between the graph and the checkpoint being loaded.
       # We add a more reasonable error message here to help users (b/110263146)
-      raise _wrap_restore_error_with_msg(
+      raise saver._wrap_restore_error_with_msg(
           err, "a mismatch between the current graph and the graph")
 
 
+def _de_keras_save_func(original_save_func,
+                        model,
+                        filepath,
+                        overwrite,
+                        include_optimizer,
```

Comment:
It would be better to add comments for each important input arguments.

Reply:
fixed